src/main/java/internetshop/dao/jdbc/ProductDaoJbdcImpl.java

```diff
@@ -0,0 +1,110 @@
+package internetshop.dao.jdbc;
+
+import internetshop.controller.LoginController;
+import internetshop.dao.ProductDao;
+import internetshop.lib.Dao;
+import internetshop.model.Product;
+import internetshop.util.conection.ConnectionUtil;
+import org.apache.log4j.Logger;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJbdcImpl implements ProductDao {
+    private static Logger logger = Logger.getLogger(LoginController.class);
+
+    @Override
+    public Product create(Product element) {
+        String query = "INSERT INTO internetshop.products (name, price) VALUES (?, ?)";
+        try ( Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, element.getName());
+            preparedStatement.setDouble(2, element.getPrice());
+            preparedStatement.executeUpdate();
+            logger.info(String.format(
+                    "Product %s with price %s has been created",
+                    element.getName(),
+                    String.valueOf(element.getPrice())));
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
let's create DataProcessingException() and throw it instead of  e.printStackTrace(); everywhere